pkg/termios/termios_test.go

```diff
@@ -28,6 +30,7 @@ func TestNew(t *testing.T) {
 }
 
 func TestRaw(t *testing.T) {
+	testutil.SkipIfNotRoot(t)
```

Comment:
Can you leave a comment that says it doesn't work on CircleCI anymore, so we're restricting it to be a VM test?

Reply:
Done